consensus/src/chained_bft/block_storage/block_tree.rs

```diff
@@ -158,38 +162,51 @@ where
         }
     }
 
+    // This method will only be used in this module.
+    fn get_linkable_block(&self, block_id: &HashValue) -> &LinkableBlock<T> {
```

Comment:
Do you mean keeping only one `get()` returning `Option`?

Reply:
No, I mean we want either `get_linkable_root` (both `get_`) or `linkable_block` (both without prefixes).

I don't think you reeeally need the `.expect()`s BTW, but they're ok for now. One thing to know to make manipulating the `Option`s easier is that you can `vec.extend(option)` since `Option` has an `Iterator`.